spec/spec_helper.rb

```diff
@@ -2,6 +2,10 @@
 
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'blueprinter'
+require 'pry'
+require 'json'
```

Comment:
Are these `require`'s necessary?

Reply:
I hope not! :)

Reply:
I added `require 'json'` here because without it many spec files fail when run individually (i.e. `bundle exec rspec spec/units/view_spec.rb`) because JSON is uninitialized. The test suite works currently when run in its entirety because of the order of the specs. When we use active record in the integration tests, which run first, it imports JSON and allows the subsequent tests to pass. Is it a bad idea to import this dependency in the spec helper?

I added `require 'pry'` simply for the convenience of debugging, but I understand if that's superfluous.